-----
std_msgs/String dev_description
std_msgs/String dev_id

